authorities.md

```diff
@@ -0,0 +1,126 @@
+# Bank Authorities API
+
+A bank authority represents an individual's consent for Centrapay to transfer funds to and from a bank account. In order to use a Centrapay wallet an individual must have a verified bank authority for that wallet.
```

Comment:
In order to move funds between a bank account and a Centrapay wallet, there must be a verified bank authority for the Centrapay account that the wallet belongs to.